python/lsst/cp/pipe/makeBrighterFatterKernel.py

```diff
@@ -156,1253 +120,308 @@ class MakeBrighterFatterKernelTaskConfig(pexConfig.Config):
         doc="The target residual error for the successive over-relaxation method",
         default=5.0e-14
     )
-    nSigmaClipKernelGen = pexConfig.Field(
-        dtype=float,
-        doc="Number of sigma to clip to during pixel-wise clipping when generating the kernel. See "
-        "the generateKernel docstring for more info.",
-        default=4
-    )
-    nSigmaClipXCorr = pexConfig.Field(
-        dtype=float,
-        doc="Number of sigma to clip when calculating means for the cross-correlation",
-        default=5
-    )
-    maxLag = pexConfig.Field(
-        dtype=int,
-        doc="The maximum lag (in pixels) to use when calculating the cross-correlation/kernel",
-        default=8
-    )
-    nPixBorderGainCalc = pexConfig.Field(
-        dtype=int,
-        doc="The number of border pixels to exclude when calculating the gain",
-        default=10
-    )
-    nPixBorderXCorr = pexConfig.Field(
-        dtype=int,
-        doc="The number of border pixels to exclude when calculating the cross-correlation and kernel",
-        default=10
-    )
-    biasCorr = pexConfig.Field(
-        dtype=float,
-        doc="An empirically determined correction factor, used to correct for the sigma-clipping of"
-        " a non-Gaussian distribution. Post DM-15277, code will exist here to calculate appropriate values",
-        default=0.9241
-    )
-    backgroundBinSize = pexConfig.Field(
-        dtype=int,
-        doc="Size of the background bins",
-        default=128
-    )
-    fixPtcThroughOrigin = pexConfig.Field(
+
+    # These are unused.  Are they worth implementing?
```

Comment:
Do we anticipate using this? What do we mean exactly by averaging vs quadratic mean? I'm trying to figure out if we will use this, or not, or if we should eliminate this option, as the comment says.

Reply:
I see the options below; it seems that these are options to produce a preKernel before using the SOR method to find the final kernel.

Reply:
I've removed the comment, as I did do those implementations.  The quadratic fit assumes that the measured correlation increases with flux.  As far as I can tell, this was added to test if the simple scale and average method was incomplete.  In the end, I updated the implementation so that if those tests were repeated, there'd be code to handle them.